### PR TITLE
feat(chromium): roll Chromium to r650583

### LINKED ISSUE #4281: Puppeteer consider visible element as not visible
Hello I have a similar issue than the one you previously corrected in ticket #4274 but this one is still present even with last version of master branch of puppeteer

## Steps to reproduce

**Tell us about your environment:**

* Puppeteer version: master branch ( 8f8e77492dd91a00d53965b998b4199bcc5e29a3 )
* Platform / OS version: Debian (Dockerized in node:10-stretch-slim)
* URLs (if applicable): https://www.edistat.com/
* Node.js version: v10.15.1

**What steps will reproduce the problem?**

Open the website
Click on `.list-group .list-group-item:nth-of-type(1) .gotodetail`
Wait for the url `https://www.edistat.com/livre_tarifs.php?ean=9782702165485`

_Please include code that reproduces the issue._
Note 1 : this code sample works fine until 1.5.0 of puppeteer and below

```
const browser = await puppeteer.launch();

const page = await browser.newPage();
await page.goto('https://www.edistat.com/');

await Promise.all([
  page.click('.list-group .list-group-item:nth-of-type(1) .gotodetail'),
  page.waitForNavigation({waitUntil: ['networkidle0', 'load', 'domcontentloaded'], timeout: 30000})
]);
console.log(await page.url());
await page.screenshot({path: 'screenshot.png', fullPage: true});

await browser.close();

```

**What is the expected result?**
The page `https://www.edistat.com/livre_tarifs.php?ean=9782702165485` should be opened when clicking on `.list-group .list-group-item:nth-of-type(1) .gotodetail`

**What happens instead?**
Browser is stuck on homepage with the message 

> Error running your code. Error: Node is either not visible or not an HTMLElement

### PR DESCRIPTION
**Pull request by aslushnikov**
This roll includes:
- https://crrev.com/650583 - DevTools: fix DOM.getContentQuads for a
  corner case

Fixes #4281.

### CODE REVIEW

**Review by JoelEinbinder — APPROVED** [review]

File: test/click.spec.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ module.exports.addTests = function({testRunner, expect}) {
       expect(await page.evaluate(() => result)).toBe('Clicked');
     });
     // @see https://github.com/GoogleChrome/puppeteer/issues/4281
-    xit('should click on a span with an inline element inside', async({page, server}) => {
+    it('should click on a span with an inline element inside', async({page, server}) => {
```
Comment (JoelEinbinder) [inline test/click.spec.js:38]: it works on firefox?

Comment (aslushnikov) [reply test/click.spec.js:38]: yes :-)
